core/http/src/known_media_types.rs

```diff
@@ -40,6 +40,7 @@ macro_rules! known_media_types {
         MOV (is_mov): "quicktime video", "video", "quicktime",
         MP4 (is_mp4): "MPEG4 Video", "video", "mp4",
         ZIP (is_zip): "ZIP archive", "application", "zip",
+        MP3 (is_mp3): "MP3 Audio", "audio", "mp3",
```

Comment:
This MIME type should be `audio/mpeg`, not `audio/mp3`.

Reply:
Why was this marked as resolved?